tests/utils/test_cli_util.py

```diff
@@ -38,7 +38,7 @@ def test_metrics_build(self):
         func_name = 'test'
         exec_date = datetime.utcnow()
         namespace = Namespace(dag_id='foo', task_id='bar', subcommand='test', execution_date=exec_date)
-        metrics = cli._build_metrics(func_name, namespace)
+        metrics = cli._build_metrics(func_name, [namespace], {})
```

Comment:
We changed the function arguments, but it's supposed to be a private function anyway.